packages/react-devtools-shared/src/backend/index.js

```diff
@@ -10,13 +10,15 @@
 import Agent from './agent';
 
 import type {DevToolsHook, RendererID, RendererInterface} from './types';
+import {isSynchronousXHRSupported} from './utils';
 
 export type InitBackend = typeof initBackend;
 
 export function initBackend(
   hook: DevToolsHook,
   agent: Agent,
   global: Object,
+  getIsReloadAndProfileSupported: () => boolean = defaultGetIsReloadAndProfileSupported,
```

Comment:
Can this be boolean with a default value `false`? And then wrappers around it are going to pass just a boolean value, and `react-devtools-core` is going to pass `getIsReloadAndProfileSupported()`.

With a current implementation, the extension is going to use `defaultGetIsReloadAndProfileSupported`, but actually it should just receive a static `true` value.